dask_mpi/tests/test_core.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import print_function, division, absolute_import
+
+import os
+import sys
+import subprocess
+
+import pytest
+pytest.importorskip('mpi4py')
+
+from distributed.utils import tmpfile
+from distributed.utils_test import loop  # noqa: F401
+
+
+def test_basic(loop):
+    script_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'basic.py')
+    with tmpfile(extension='json') as fn:
+        with open('test_basic.out', 'w') as fo:
```

Comment:
Is there a reason for not using tmpfile here too?

Reply:
Oh.  That was just for diagnostics.  (I assume you are referring to the `test_basic.out` file.)  I need to remove that entirely, frankly.  Sorry about that!

Reply:
Ok.  I've removed the `test_basic.out` file with commit https://github.com/dask/dask-mpi/pull/6/commits/2f26b631eab6702f5a3b3f63bb755b45eae4737c.